packages/internal-postcss-fix-relative-paths/src/index.ts

```diff
@@ -9,12 +9,12 @@ export default function fixRelativePathsPlugin(): Plugin {
   let touched: WeakSet<AtRule> = new WeakSet()
 
   function fixRelativePath(atRule: AtRule) {
-    let rootPath = getRoot(atRule)?.source?.input.file
+    let rootPath = atRule.root().source?.input.file
```

Comment:
I simplified this. `Node` has a `root()` method that walks up the tree for you.

It also stops at Root instead of a Document (an optional parent node of Root)